[DOCFIX] Add detailed steps in the upgrade document (#15972)
Alluxio/alluxio: docs/en/operation/Upgrade.md

so pre-2.0.0 clients do not work with post-2.0.0 servers, and vice-versa.
Upgrade all applications to use the alluxio-2.x client.

Please refer to the following steps：
1. Back up the metadata of the files in Alluxio. Please refer to the [documentation]({{ '/en/operation/Admin-CLI.html' | relativize_url }}#backup) on `backup` command.
2. Stop the Alluxio cluster
```console
$ ./bin/alluxio-stop.sh all
```
3. Update the Alluxio client jar path for all your applications. For example, `Yarn`, `Spark`, `Hive` and `Presto`.

> eg.

Reviewer: nit: no need to start a newline

so pre-2.0.0 clients do not work with post-2.0.0 servers, and vice-versa.
Upgrade all applications to use the alluxio-2.x client.

Please refer to the following steps：

> 1. Back up to the default backup folder `/alluxio_backups` of the root under storage system. This default backup directory can be configured by setting `alluxio.master.backup.directory`.

Reviewer: ```suggestion
1. Back up to the default backup folder `/alluxio_backups` in the root under storage system. This default backup directory can be configured by setting `alluxio.master.backup.directory`.
```

```
$ ./bin/alluxio fsadmin backup
```
2. Stop the alluxio cluster
```
$ ./bin/alluxio-stop.sh all
```

> 3. Change the symlink of the client jar for the following components if you are using CDH or CDP：`CM` `CDH` `ZooKeeper` `HDFS` `Yarn` `Spark` `Hive`

Reviewer: You can easily miss something if you try to list them all.
```suggestion
3. Update the Alluxio client jar path for all your applications. For example, `Yarn`, `Spark`, `Hive` and `Presto`.
```

>    <br />eg.

Reviewer: is this `br` necessary?

Reviewer 2: +1, please remove these `br` tags.

>    <br />In the "YARN (MR2 Included)" section of the Cloudera Manager, in the "Configuration" tab, search for the parameter "Gateway Client Environment Advanced Configuration Snippet (Safety Valve) for hadoop-env.sh". Then add the following line to the script:

Reviewer: Instead of a verbal instruction, can you paste a link to the doc or use a screenshot? In doc pages like https://docs.alluxio.io/os/user/stable/en/Overview.html you can find how images are added to the doc.

   ```shell

>    HADOOP_CLASSPATH=/path/to/alluxio/client/alluxio-enterprise-2.8.0-3.0-client.jar:${HADOOP_CLASSPATH} 

Reviewer: You are adding to the OS doc so use the OS client jar name

   ```
4. Start the alluxio cluster
```
$ ./bin/alluxio-start.sh all
```

> 5. Please restart all the following components integrated with alluxio if you are using CDH or CDP：`CM` `CDH` `ZooKeeper` `HDFS` `Yarn` `Spark` `Hive`

Reviewer: Similarly, `If you have updated the Alluxio client jar for an application, restart that application to use the new Alluxio client jar.`

so pre-2.0.0 clients do not work with post-2.0.0 servers, and vice-versa.
Upgrade all applications to use the alluxio-2.x client.

Please refer to the following steps：

> 1. Back up to the default backup folder `/alluxio_backups` in the root under storage system. This default backup directory can be configured by setting `alluxio.master.backup.directory`.

Reviewer 2: Better leave the details to a link to an existing section on this `backup` command:
https://docs.alluxio.io/ee/user/stable/en/operation/Admin-CLI.html#backup

```suggestion
1. Back up the metadata of the files in Alluxio. Please refer to the [documentation](https://docs.alluxio.io/ee/user/stable/en/operation/Admin-CLI.html#backup) on `backup` command.
```

```
$ ./bin/alluxio fsadmin backup

> ```

Reviewer 2: remove this as it is covered in the doc of `backup` command.

> 2. Stop the alluxio cluster

Reviewer: ```suggestion
2. Stop the Alluxio cluster
```

```
$ ./bin/alluxio-stop.sh all
```
3. Update the Alluxio client jar path for all your applications. For example, `Yarn`, `Spark`, `Hive` and `Presto`.
   <br />eg.
   <br />In the "YARN (MR2 Included)" section of the Cloudera Manager, in the "Configuration" tab, search for the parameter "Gateway Client Environment Advanced Configuration Snippet (Safety Valve) for hadoop-env.sh". Then add the following line to the script:
   ```shell

>    HADOOP_CLASSPATH=/path/to/alluxio/client/lluxio-2.8.0-client.jar:${HADOOP_CLASSPATH} 

Reviewer 2: Use a variable for client jar path, so that the version number is always up to date.
See example here https://docs.alluxio.io/os/user/stable/en/compute/HBase.html#distribute-the-alluxio-client-jar

```suggestion
   HADOOP_CLASSPATH={{site.ALLUXIO_CLIENT_JAR_PATH}}:${HADOOP_CLASSPATH} 
```

   ```

>    It should look something like this:

Reviewer: ```suggestion
   It should look like:
```

   ![locality]({{ '/img/screenshot_cdh_compute_hadoop_classpath.png' | relativize_url }})
4. Start the alluxio cluster

> ```

Reviewer 2: ```suggestion
```console
```
here and above

Upgrade all applications to use the alluxio-2.x client.

Please refer to the following steps：

> 1. Back up the metadata of the files in Alluxio. Please refer to the [documentation](https://docs.alluxio.io/ee/user/stable/en/operation/Admin-CLI.html#backup) on `backup` command.

Reviewer 2: I forgot to mention the url needs to use a variable and `relativize_url`.
